cinnamon-dynamic-wallpaper@TobiZog/README.md

```diff
@@ -1,33 +1,42 @@
 # Cinnamon Dynamic Wallpaper
+<img src="cinnamon-dynamic-wallpaper@TobiZog/5.4/icons/icon.svg" alt="drawing" width="200" style="margin-left:auto; margin-right:auto; width:50%; display:block"/>
```

Comment:
This should be an absolute link. A relative link will break on something like the Spices website.

Reply:
I've removed it